packages/ckb-types/index.d.ts

```diff
@@ -81,20 +81,36 @@ declare namespace CKBComponents {
     index: number
   }
 
+  export interface Witness {
+    data: string[]
+  }
+
   /**
-   * @typedef Transaction, transaction object
+   * @typedef RawTransaction, raw transaction object
```

Comment:
We might not need a one-to-one mapping structure/class to CKB's `RawTransaction`, which is not a public struct. Having a `Transaction` with witness field but don't submit/pass that to `sendTransaction` seems sufficient.

Reply:
Well, I will find a better expression.